aws-datastore/src/test/java/com/amplifyframework/datastore/syncengine/OrchestratorTest.java

```diff
@@ -122,7 +121,7 @@ public void setup() throws AmplifyException {
     @Test
     public void itemsPlacedInStorageArePublishedToNetwork() throws AmplifyException {
         // Arrange: orchestrator is running
-        orchestrator.start();
+        orchestrator.start().subscribe();
```

Comment:
maybe `TestObserver<Void> observer = orchestrator.start().test()`? I think those auto-dispose, which is cool.